internal/info/history.go

```diff
@@ -17,24 +21,23 @@ func (i *InfoBuf) LoadHistory() {
 	if config.GetGlobalOption("savehistory").(bool) {
 		file, err := os.Open(filepath.Join(config.ConfigDir, "buffers", "history"))
 		var decodedMap map[string][]string
-		if err == nil {
-			defer file.Close()
-			decoder := gob.NewDecoder(file)
-			err = decoder.Decode(&decodedMap)
-
-			if err != nil {
-				i.Error("Error loading history:", err)
-				return
+		if err != nil {
+			if !errors.Is(err, fs.ErrNotExist) {
+				i.Error("Error:", err)
```

Comment:
I think it's better to keep `Error loading history:`, to make it clear what is this error about.

Also (rather unfortunately) we should add a space after the colon (here and elsewhere in `i.Error()`), since `i.Error()` uses `fmt.Sprint()` which [only adds spaces between operands when neither is a string](https://pkg.go.dev/fmt#Sprint).

Reply:
> Also (rather unfortunately) we should add a space after the colon (here and elsewhere in `i.Error()`), since `i.Error()` uses `fmt.Sprint()` which [only adds spaces between operands when neither is a string](https://pkg.go.dev/fmt#Sprint).

Wouldn't it be better to align it via using `fmt.Sprintln()` instead of `fmt.Sprint()` in `func (i *InfoBuf) Message(msg ...interface{})` and `func (i *InfoBuf) Error(msg ...interface{})`?

Reply:
We don't want `i.Msg` to include newline?

Reply:
Indeed, might cause problems in the infobar.